x-pack/test/functional/services/observability/alerts/common.ts

```diff
@@ -192,7 +198,6 @@ export function ObservabilityAlertsCommonProvider({
   const viewRuleDetailsLinkClick = async () => {
     return await (await testSubjects.find(VIEW_RULE_DETAILS_FLYOUT_SELECTOR)).click();
   };
-
   // Workflow status
   const setWorkflowStatusForRow = async (rowIndex: number, workflowStatus: WorkflowStatus) => {
```

Comment:
@ersin-erdal I see you removed workflow status related code. Do we still need this? Can we remove?

Reply:
This might require bigger refactoring, since this function might be used in other files as well. I live it up to you to decide if you want to do this refactoring as part of this PR

Reply:
Yes i tried to get rid of it but a lot of functional tests failed! then i put it back.
It requires its own issue...